### PR TITLE
Add rollback attributes on transition of all admin routes

### LINKED ISSUE #585: Rollback the model attributes on route transition on all admin routes
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

The attributes are not rolled back currently upon transitioning route. Infact in admin settings route, some routes share a common model, and the save action of a single route serves as save for all of them, which is  non-intuitive.
**Expected behavior:**

The attributes should be rolled back upon transitioning away.

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds rollback of attributes on transition of all admin routes

Fixes #585

### CODE REVIEW

File: app/routes/admin/content/index.js [hunk 1]
```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('store').queryRecord('setting', {});
+  },
+  actions: {
+    willTransition() {
+      this.get('controller.model').rollbackAttributes();
```
Comment (niranjan94) [inline app/routes/admin/content/index.js]: Why ? Isn't `model` directly accessible ? 🤔

Comment (CosmicCoder96) [inline app/routes/admin/content/index.js]: @niranjan94 Of course sorry, was testing something and pasted it everywhere.  I will change it

**Comment by CosmicCoder96** [comment]
@niranjan94 Done !

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

File: app/routes/admin/content/index.js (hunk 1 above)
Comment (CosmicCoder96) [inline app/routes/admin/content/index.js]: @niranjan94 That resulted in an error, I didn't knew it was because of that, please check the comments below.

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@CosmicCoder96 One way of accessing the model in the route is by using 
`this.get('currentModel')`

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

File: app/routes/admin/content/index.js [hunk 2]
```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('store').queryRecord('setting', {});
+  },
+  actions: {
+    willTransition() {
+      this.get('currentModel').rollbackAttributes();
```
Comment (niranjan94) [inline app/routes/admin/content/index.js]: Mmm ... actually ... keep your previous one. `controller.model`. `currentModel` is a private property and could change in future.

Ref: https://github.com/emberjs/ember.js/issues/11760

**Comment by CosmicCoder96** [comment]
@geekyd Thanks ! @niranjan94  I have changed it to what he suggested. I always wrongly assumed model is available for a route via `this.get('model')` Please review

File: app/routes/admin/content/index.js (hunk 2 above)
Comment (CosmicCoder96) [inline app/routes/admin/content/index.js]: 😅 okay

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

**Comment by CosmicCoder96** [comment]
@niranjan94 Done !

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-586-open-event-frontend-fossasia.surge.sh

**Pull request merged**